src/mono/wasm/runtime/exports-linker.ts

```diff
@@ -35,7 +35,7 @@ const mono_wasm_threads_exports = !MonoWasmThreads ? undefined : {
     // mono-threads-wasm.c
     mono_wasm_pthread_on_pthread_attached,
     // threads.c
-    mono_wasm_eventloop_has_unsettled_interop_promises,
+    mono_wasm_eventloop_has_unsettled_interop_promises: monoWasmEventLoopHasUnsettledInteropPromises,
```

Comment:
I would prefer to keep the names consistent across languages. 
so rename also C code of `mono_wasm_eventloop_has_unsettled_interop_promises` or keep the original name.

The important thing here is that fulltext search should find both, usage and definitionin both languages.

Reply:
I am voting for keeping the original name then.

Reply:
in this case it's C import. should the same rule apply ? Or the other ?

Reply:
I would keep even imports to C in C style convention

Reply:
ok